crates/cargo-contract/src/cmd/mod.rs

```diff
@@ -230,3 +233,12 @@ pub fn basic_display_format_contract_info(info: &ContractInfo) {
         MAX_KEY_COL_WIDTH
     );
 }
+
+/// Display all contracts addresses in a formatted way
+pub fn display_all_contracts(contracts: &[<DefaultConfig as Config>::AccountId]) {
+    contracts
+        .iter()
+        .for_each(|e: &<DefaultConfig as Config>::AccountId| {
+            name_value_println!("Contract", format!("{}", e), MAX_KEY_COL_WIDTH);
+        })
+}
```

Comment:
I would rewrite it to display the result something like:
```bash
Found contracts:
    * <Address>
    * <Address>
    * <Address>
 ```

Reply:
I think that in case when we have more than 1000 contracts, it will be hard to see this line Found contracts at the log beginning. I would opt to keep it as is. What do you think?

Reply:
Then I would enumerate them at least. Useful when you want to pipe the output into another bash command:
```bash
Found contracts:
    1. <Address>
    2. <Address>
    ...
    1000. <Address>
```
It is up to you if you want to keep `Found contracts:` or not.

Reply:
> Useful when you want to pipe the output into another bash command

In this case it is best then to have *only* the list of contract addresses so instead of

```
        Contract 5DpyotvcUDkFxTtHzkwR7Rqe37bdjc6M5Rxeiy57Gv2RQzp8
        Contract 5GsoMKLTgFEpzHQTi7c4ZKCLZC6VxtHZxwMydSi79uemABez
```
have
```
5DpyotvcUDkFxTtHzkwR7Rqe37bdjc6M5Rxeiy57Gv2RQzp8
5GsoMKLTgFEpzHQTi7c4ZKCLZC6VxtHZxwMydSi79uemABez
```
I like it this way, unix command compatible.

Reply:
Agreed

Reply:
Done